js/narrativeView.js

```diff
@@ -241,9 +241,13 @@ define([
             var numberOfItems = this.model.get('_children').length;
 
             if ($(event.currentTarget).hasClass('narrative-control-right')) {
-                this.model.setActiveItem(++stage);
+                ++stage;
             } else if ($(event.currentTarget).hasClass('narrative-control-left')) {
-                this.model.setActiveItem(--stage);
+                --stage;
+            }
+
+            if (stage >= 0 && stage < numberOfItems) {
+                this.model.setActiveItem(stage);
             }
```

Comment:
Current coding style favours returning where possible i.e.
```suggestion
            if (stage < 0 || stage >= numberOfItems) return;

            this.model.setActiveItem(stage);
```
Or you could do this
```suggestion
            stage = Math.min(Math.max(stage, 0), numberOfItems);
            this.model.setActiveItem(stage);
```
Slightly surprised to find that underscore doesn't have a `clamp` function like lodash does

Is it worth moving this check to be in `setActiveItem` so that this fix would apply to anything that uses ItemsComponentModel...?